tests/sentry/api/endpoints/test_organization_events_trends_v2.py

```diff
@@ -0,0 +1,268 @@
+            response = self.client.get(
+                self.url,
+                format="json",
+                data={
+                    "end": iso_format(self.now - timedelta(minutes=1)),
+                    "start": iso_format(self.now - timedelta(hours=4)),
+                    "interval": "1h",
+                    "field": ["project", "transaction"],
+                    "query": "",
+                },
+            )
+
+        assert response.status_code == 200, response.content
+        assert response.data == []
+
+    @mock.patch("sentry.api.endpoints.organization_events_trends_v2.get_trends")
+    def test_simple_with_trends(self, mock_get_trends):
+        mock_trends_result = [
+            {
+                "project": self.project.slug,
+                "transaction": "foo",
+                "change": "regression",
+                "trend_difference": -15,
+                "trend_percentage": 0.88,
+            }
+        ]
+        mock_get_trends.return_value = {"data": mock_trends_result}
+
+        with self.feature(self.features):
+            response = self.client.get(
+                self.url,
+                format="json",
+                data={
+                    "end": iso_format(self.now),
+                    "start": iso_format(self.now - timedelta(days=1)),
+                    "interval": "1h",
+                    "field": ["project", "transaction"],
+                    "query": "event.type:transaction",
+                    "project": self.project.id,
+                },
+            )
+
+        assert response.status_code == 200, response.content
+
+        events = response.data["events"]
+        result_stats = response.data["stats"]
+
+        assert len(events["data"]) == len(mock_trends_result)
+        assert events["data"] == mock_trends_result
+
+        assert len(result_stats) > 0
+        assert len(result_stats.get(f"{self.project.slug},foo", [])) > 0
+
+    @mock.patch("sentry.api.endpoints.organization_events_trends_v2.get_trends")
+    def test_simple_with_no_trends(self, mock_get_trends):
+        mock_trends_result = []
+        mock_get_trends.return_value = {"data": mock_trends_result}
+
+        with self.feature(self.features):
+            response = self.client.get(
+                self.url,
+                format="json",
+                data={
+                    "end": iso_format(self.now),
+                    "start": iso_format(self.now - timedelta(days=1)),
+                    "interval": "1h",
+                    "field": ["project", "transaction"],
+                    "query": "event.type:transaction",
+                    "project": self.project.id,
+                },
+            )
+
+        assert response.status_code == 200, response.content
+
+        events = response.data["events"]
+        result_stats = response.data["stats"]
+
+        assert len(events["data"]) == len(mock_trends_result)
+        assert events["data"] == mock_trends_result
```

Comment:
nit: We shouldn't reuse the variable like this since it worsens readability
```suggestion
        assert len(events["data"]) == 0
        assert events["data"] == []
```

Reply:
done!